python/ray/train/tests/test_torch_checkpoint.py

```diff
@@ -33,6 +36,18 @@ def test_from_state_dict():
     assert actual_state_dict == expected_state_dict
 
 
+def test_no_encoding_for_dir_checkpoints(tmpdir):
+    tmpdir = Path(tmpdir)
+    with (tmpdir / "test_file").open("w"):
+        pass
+
+    # Make sure we do not double encode.
+    with patch("torch.save") as save_mock:
+        checkpoint = TorchCheckpoint.from_directory(tmpdir)
+        checkpoint.__getstate__()
+    save_mock.assert_not_called()
```

Comment:
Does this need to be captured in the context?
```suggestion
        save_mock.assert_not_called()
```

Reply:
no, it doesn'!